pkgs/os-specific/linux/kernel/manual-config.nix

```diff
@@ -114,10 +114,6 @@ let
         ++ optional (lib.versionAtLeast version "5.2" && lib.versionOlder version "5.4") ./gen-kheaders-metadata.patch;
 
       prePatch = ''
```

Comment:
This should be done in postPatch because it can break upstream patches.

Reply:
I _think_ this is intentional, but I don't understand it enough to touch it right now.

Reply:
not sure about the rest but this bit needs to be done pre

https://github.com/NixOS/nixpkgs/commit/e9a1ba099fb3bf722fdc1bb8ddc1998ff0ec830a

Reply:
If a patch changes the shebang in scripts/ld-version.sh or a line below it, it will fail to apply. A better approach would be to conditionally patch the shebang or allow it to fail.

Reply:
Let's not change the semantics of what is basically out of scope of the changes.

If you believe it should be done, you're invited to open a new PR aimed at cleaning up the nits from the derivation.